README.md

```diff
@@ -39,7 +39,7 @@ In the training phase, the best case is that we are able to detect between Y+D a
 
 The labeling technical details are described [in this issue](https://github.com/marco-c/autowebcompat/issues/2).
 
-### Training
+### Training methodologies
```

Comment:
Don't change this, it's unrelated to the PR.

Reply:
I thought "training" would be redundant and so i changed it . I'll revert it back